omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -64,7 +64,7 @@
 	        }
 	        return appList;
 		}
-	
```

Comment:
I thought you would not touch this file at all.

Reply:
I reverted it to what it was before after making changes earlier. But let me take it back to what it was on the master branch.